lib/recipient.rb

```diff
@@ -0,0 +1,55 @@
+require "httparty"
+require "awesome_print"
+require "dotenv"
+Dotenv.load
+
+class Recipient
```

Comment:
You'll notice the code for this class is pretty repetitive.  That's a clear sign that you can refactor this method to simplify the logic a bit.  

Check out the instructor reference design and ask me questions if you have them.